kedro-datasets/pyproject.toml

```
@@ -194,16 +194,15 @@ test = [
     "networkx~=2.4",
     "opencv-python~=4.5.5.64",
     "openpyxl>=3.0.3, <4.0",
-    "pandas-gbq>=0.12.0, <0.18.0; python_version < '3.11'",
-    "pandas-gbq>=0.18.0; python_version >= '3.11'",
-    "pandas~=1.3",  # 1.3 for read_xml/to_xml
+    "pandas-gbq>=0.12.0",
+    "pandas~=2.0",
```

```suggestion
    "pandas>=1.3",
```

(unless there's a reason not to)